deploy/prod.properties

```diff
@@ -1,3 +1,4 @@
 prod-server=ea-tracker.demo.marklogic.com
 user=admin
 password=
+app-port=70
```

Comment:
80 was default but we thought 80 might be being used by something else so chose 70 arbitrarily